src/components/date-picker/DatePicker.tsx

```diff
@@ -276,12 +287,33 @@ class DatePicker extends Component<
         },
       ],
     });
+
+
+    if (menuShouldBlockScroll) {
+      const scrollContainer = getScrollParent(refContainer);
+      if (scrollContainer) {
+        scrollContainer.addEventListener('touchmove',this.handleScrollParent);
+        scrollContainer.addEventListener('DOMMouseScroll',this.handleScrollParent);
+        scrollContainer.addEventListener('mousewheel',this.handleScrollParent);
+      }
+    }
   }
 
   unmountDayPickerInstance() {
+    const { menuShouldBlockScroll } = this.props;
+    const { refContainer } = this.state;
+
     if (this.dayPickerInstance) {
       this.dayPickerInstance.destroy();
       Reflect.deleteProperty(this, 'dayPickerInstance');
+      if (menuShouldBlockScroll) {
+        const scrollContainer = getScrollParent(refContainer);
+        if (scrollContainer) {
+          scrollContainer.removeEventListener('touchmove',this.handleScrollParent);
```

Comment:
FYI, React-Select uses the lib https://www.npmjs.com/package/react-scrolllock
See commit: https://github.com/JedWatson/react-select/commit/d2eea09e5bd046a9481513a68c84d733c5ea9644
This library also handles mobile devices. This could be a good solution to use it. What do you think about it?

Reply:
Thank you for sharing,
react-scrolllock is made to block the scroll on <body>, but I am blocking the scroll of a specific div